client/app/scripts/superdesk-users/views/edit-form.html

```diff
@@ -25,10 +25,7 @@
               <span sd-switch ng-model="$parent._active" ng-click="toggleStatus(_active)"></span>
             </div>
         </div>
-        <div class="error-holder">
-          <p class="alert alert-error" ng-show="error && !error.message" translate>Please fix the issues below.</p>
-          <p class="alert alert-error" ng-show="error && error.message" translate>{{error.message}}</p>
```

Comment:
The other is shown all the time as notification, so to delete both line seams to be ok.